public/components/Campaign/Campaign.js

```diff
@@ -62,17 +76,18 @@ class Campaign extends React.Component {
                         latestAnalyticsForCampaign={this.props.latestAnalyticsForCampaign} />
 
           <CampaignPerformanceBreakdown campaign={campaign}
-                        latestAnalyticsForCampaign={this.props.latestAnalyticsForCampaign} />
+                        latestAnalyticsForCampaign={this.props.latestAnalyticsForCampaign}
+                        territory={this.props.territory} />
 
           <CampaignEdit campaign={campaign}
                         updateCampaign={this.props.campaignActions.updateCampaign}
                         saveCampaign={this.props.campaignActions.saveCampaign}/>
-          <CampaignAnalytics campaign={campaign} />
+          {/*}<CampaignAnalytics campaign={campaign} /> */}
```

Comment:
What's this for?

Reply:
I'm commenting out until we also make use of the territory specific data but I wanted to do this in a seperate PR.